projects/longevity/pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[tool.pinto]
+steps = [
+    #"data_gen:launch-datagen",
+    #"training:launch-training",
+    #"exporting:launch-export:resnet",
+    "inference:launch-inference",
+]
+
+
+[tool.typeo.base]
+
+Tb = 86400 #5184000  # 60 days of background
+shifts = [0, 1]
+
+inference_psd_length = 64
+training_psd_length = 8
+inference_sampling_rate = 16
```

Comment:
You can lower this to 4 Hz, though it would be nice if there were a general way to enforce consistency across configs